### ISSUE TITLE
Documentation: RevisionToken flow & security implications

### ISSUE THREAD

**Issue by nmiodice** [google/exposure-notifications-server#784]
### TL;DR

During an internal review of the `RevisionToken` feature recently introduced to the service code, we identified a risk related to the public perception of this feature. There is clearly a lot of effort put into making it difficult (impossible?) to associate multiple client requests over time, and it would be great to have this documented in some form of public documentation, likely in this repository or [here](https://developers.google.com/android/exposure-notifications/verification-system).

### Design

**Proposal**
I'm happy to share some documentation we wrote internally (lets connect on Slack) but I don't think it is a good fit for the public facing documentation.

**Alternatives considered**
N/A

**Resources**
N/A

**Additional information**
N/A

**Comment by mikehelmick** [comment]
I submitted an internal change to update
https://developers.google.com/android/exposure-notifications/verification-system
